scripts/build_workflow_release.sh

```diff
@@ -2,7 +2,7 @@
 set -e -o pipefail
 
 declare -r SCRIPT_DIR=$(cd $(dirname $0) && pwd)
-declare -r DSDE_PIPELINES_ROOT=$(cd $(dirname ${SCRIPT_DIR}) && pwd)
+declare -r WARP_ROOT=$(cd $(dirname ${SCRIPT_DIR}) && pwd)
```

Comment:
You could name it `REPO_ROOT` in case we move everything again.

Reply:
I will change this